tests/bluetooth/host/keys/bt_keys_add_type/src/main.c

```diff
@@ -0,0 +1,122 @@
+/*
```

Comment:
Out of curiosity, why is the most recent parent commit for these tests from 2016? What version of the code is being targeted? A Bluetooth host from 2016 is not necessarily qualifiable for new designs (it may be, but it's a risk).